packages/faustwp-cli/src/healthCheck/validateFaustEnvVars.ts

```diff
@@ -28,4 +28,28 @@ export const validateFaustEnvVars = () => {
       'Please make sure your production Faust app runs with a WordPress instance on https!',
     );
   }
+  if (getWpSecret()) {
+    // send secret key
+    const apiUrl = `${getWpUrl()}/wp-json/faustwp/v1/validate_secret_key`;
+    const headers = {
+      'x-faustwp-secret': getWpSecret() || '',
+    };
+    try {
+      const response = await fetch(apiUrl, {
+        headers,
+        method: 'POST',
+        timeout: 30000, // 30 seconds timeout
+      } as unknown as RequestInit);
+      if (response.status === 204) {
+        // Success: User receives a 204 status code
+      } else if (response.status === 401) {
+        // Unauthorized: User receives a 401 status code AND the message below
+        warnLog(
+          'Check to ensure your FAUST_SECRET_KEY matches your Faust Secret Key under wp-admin settings',
+        );
+      }
+    } catch (error) {
+      console.log('error', error);
```

Comment:
Since this will be a user facing error in the CLI, you may want to use `errorLog`